miniserver/utils.js

```diff
@@ -0,0 +1,81 @@
+const JSSHA = require('../server/utils')
```

Comment:
I think we can just include the file from /server rather than creating a duplicate